### ISSUE TITLE
Multi sensors with iotivity-constrained not stable

### ISSUE THREAD

**Issue by uhofemei** [01org/SmartHome-Demo#136]
The multi sensor solution on Arduino 101 doesn't run stable. I'm able to access temperature, ambient light and motion for 2 minutes or less before the resources disappear. I have reason to believe this is caused by Zephyr crashing and not the BLE interface going down. This is for the reason that our "other" BLE solution being the power meter runs stable after the multi sensor solution has gone down.

**Comment by uhofemei** [comment]
UPDATE: James Prestwood has multiple resources with ZJS working. Using ZJS we will integrate the temperature and ambient light sensor. This leaves RGB led and motion for the native implementation.

**Comment by uhofemei** [comment]
bluetoothd[766]: Unable to register GATT service with handle 0x000b for device F1:13:D1:99:72:CD

**Comment by gvancuts** [comment]
Still experiencing some instabilities when running multiple Zephyr-based sensors at the same time.

Set-up:
- Gateway: Ubuntu 17.04 + iot-rest-api-server (installed directly)
- 1x Power Meter
- 1x multi-sensor (native Zephyr + iotivity-constrained)
- 1x ZJS sensor

It has been observed that using more powerful power supplies improves the BLE stability. @uhofemei will also test reverting the Gateway set-up to Ubuntu 16.04 LTS (which is still the most commonly used environment for testing).

**Comment by uhofemei** [comment]
Using Ubuntu 16.04 made no difference. Swapping hardware made no difference. I have to note that it seems to be required to establish a BLE connection with the device within <=10 seconds after it was powered up or the connection won't get established. Once the device is up, it either remains stable for up to 48 hours (tested) or it goes down within 3 minutes without accessing any of the OCF resources. All I've been running to test this is a native OCF client triggering a discovery.

**Comment by gvancuts** [comment]
Is this now a duplicate of #140 ?

**Comment by uhofemei** [comment]
Yes, I would say so. I also filed a bug in the ZJS issue tracker. https://github.com/01org/zephyr.js/issues/1237 

Geoff was telling me they will re-direct this to Kishen for further investigation. The isuse opened in the iotivity-constrained repo that discusses this is: https://github.com/iotivity/iotivity-constrained/issues/5
